wordpress-mobile/WordPress-Android
# Issue/13509 fix check box clicks

Fixes #13509 


To test:
1. Launch App
2. Set up the Activity Log Filters feature flag
3. Navigate to ActivityLog
4. Tap on the ActivityType chip
5. Tap on the checkbox proper - not the row - select mutliples
6. Tap clear
7. Notice that the checkboxes are cleared
8. Tap on the checkbox proper - not the row - select mutliples
9. Tap on Apply
10. Tap on the ActivityType chip
11. Notice that previously selected items are checked 

PR submission checklist:
